08_modules/README.md

```diff
@@ -0,0 +1,9 @@
+Invoke the following command to generate the images of the houses:
+
+~~~ shell
+sbt run
+~~~
+
+This will create a file houses.pdf.
```

Comment:
I am not sure this README is the correct one? The program that creates the houses comes earlier in the curriculum IIRC (not in module 8)

Reply:
you're right!